docs/src/extend/custom-parsers.md

```diff
@@ -74,8 +65,45 @@ interface Token {
 
 The range indexes of all tokens and comments must not overlap with the range of other tokens and comments.
 
-### The `Literal` node:
+### The `Literal` Node
 
 The `Literal` node must have `raw` property.
 
 * `raw` (`string`) is the source code of this literal. This is the same as `code.slice(node.range[0], node.range[1])`.
+
+## Packaging a Custom Parser
+
+TODO: Add info on turning into a package
```

Comment:
to reviewers, would you mind providing some high level direction on what should be included here? perhaps a simple example with a bit of description?

Reply:
Hehe, didn't realize we left a TODO in there.

Basically, what this section should be is how to create an npm package out of your custom parser. I imagine it would simply be:

1. Create your npm package
2. Ensure that `main` in `package.json` points to your parser file
3. Publish your npm package
4. Show how to use the npm package in a config file